Placeeo/Controller/MapViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  MapViewController.swift
+//  Placeeo
+//
+//  Created by Pasha Berger on 13.09.2021.
+//
+
+import UIKit
+import MapKit
+import CoreLocation
+
+class MapViewController: UIViewController, MKMapViewDelegate {
+        
+    @IBOutlet weak var mapView: MKMapView!
+    
+    var locationManager : CLLocationManager!
+    
+    
+    
+    let place = Place()
+    var placeArray = [Place]()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+       
+
+        
+    }
+    override func viewDidAppear(_ animated: Bool) {
+        determineCurrentLocation()
+    }
+    
+    
+    
+    
+    
+    func show(places: [Place]){
+        
+    }
+    
+    func clearPlaces() {
+       
+    }
+    
+}
+    //MARK: - CLLocation Manager Delegate
+    extension MapViewController : CLLocationManagerDelegate {
+        func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+            let mUserLocation:CLLocation = locations[0] as CLLocation
+            
+            let center = CLLocationCoordinate2D(latitude: mUserLocation.coordinate.latitude, longitude: mUserLocation.coordinate.longitude)
+            let mRegion = MKCoordinateRegion(center: center, span: MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.01))
+            mapView.setRegion(mRegion, animated: true)
```

Comment:
These lines are actually showing the place on a map, so let's move them to the `show(places:` method

Reply:
Or you can create a new method `show(place: Place)` which will show one spot on a map, what these lines actually do